lib/anubis.go

```diff
@@ -126,25 +129,30 @@ func New(opts Options) (*Server, error) {
 	mux := http.NewServeMux()
 	xess.Mount(mux)
 
-	mux.Handle(anubis.StaticPath, internal.UnchangingCache(internal.NoBrowsing(http.StripPrefix(anubis.StaticPath, http.FileServerFS(web.Static)))))
+	// Helper to add global prefix
+	registerWithPrefix := func(pattern string, handler http.Handler, method string) {
+		if method != "" {
+			method = method + " " // methods must end with a space to register with them
+		}
+		prefix := method + anubis.BasePrefix
+		mux.Handle(prefix+pattern, handler)
+	}
+
+	registerWithPrefix(anubis.StaticPath, internal.UnchangingCache(internal.NoBrowsing(http.StripPrefix(anubis.BasePrefix+anubis.StaticPath, http.FileServerFS(web.Static)))), "")
```

Comment:
We are going to need to make syntax sugar for this at some point lol, it's fine as it is now though.

Reply:
Do you want me to revert it then or leave how I changed it then? Not 100% sure what you mean.